draft-ietf-scone-protocol.md

```diff
@@ -473,22 +473,16 @@ from the network element with the lower policy.
 
 The following pseudocode indicates how a network element might detect a SCONE
 packet and replace an existing rate signal,
-given throughput advice (`target_throughput`).
+with an encoded throughput advice (`target_signal`).
 
```

Comment:
Doesn't read very well. Suggestion: "... replace an existing rate signal (`packet_signal`) with a new rate signal encoding the throughput advice of this network element (`target_signal`)."